wsdl/ver20/media/wsdl/media.wsdl

```diff
@@ -1300,6 +1327,208 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 				</xs:complexType>
 			</xs:element>
 			<!--===============================-->
+			<!--============AudioClipManagement===================-->
+			<xs:complexType name="AudioClip">
+				<xs:sequence>					
+					<xs:element name="Enabled" type="xs:boolean">
+						<xs:annotation>
+							<xs:documentation> Enables/disables the audio clip.</xs:documentation>
+						</xs:annotation>
+					</xs:element>
+					<xs:element name="Name" type="xs:string">
+					<xs:annotation>
+						<xs:documentation> Name of the audio clip.</xs:documentation>
+					</xs:annotation>
+					</xs:element>
+					<xs:element name="AudioOutputToken" type="tt:ReferenceToken" minOccurs="0" maxOccurs="unbounded">
+					<xs:annotation>
+						<xs:documentation> Optional Token list of physical Audio Outputs. </xs:documentation>
+					</xs:annotation>
+					</xs:element>
+					<xs:element name="Type" type="xs:string">
+					<xs:annotation>
+						<xs:documentation> Format of the audio clip. For definitions see tt:SupportedAudioClipFormat parameter.</xs:documentation>
```

Comment:
Looks like tt:SupportedAudioClipFormat is not defined, can we define something like below, 
<xs:simpleType name="SupportedAudioClipFormat ">
				<xs:restriction base="xs:string">					
					<xs:enumeration value="WAV"/>
					<xs:enumeration value="MP3"/>
					<xs:enumeration value="OGG-Opus"/>
					<xs:enumeration value="OGG-Vorbis"/>
					<xs:enumeration value="OGG-Flac"/>
					<xs:enumeration value="OGG-Speex"/>
				</xs:restriction>
</xs:simpleType>

Reply:
PR is updated with SupportedAudioClipFormat enumerations as suggested.